qemu/tests/sr_iov_hotplug.py

```diff
@@ -170,7 +212,7 @@ def _find_pci():
 
             # Assign static IP to the hotplugged interface
             if params.get("assign_static_ip", "no") == "yes":
-                cmd = []
+                cmd = "service networking restart"
```

Comment:
@xiywang In L 233, I have handled for it as there is a check for a distro other than Ubuntu.